internal/api/group.go

```diff
@@ -110,10 +110,18 @@ func handleCreateGroup(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	err = createGroupRequest.MattermostEnv.Validate()
```

Comment:
nit: would like to see all validation move to a `Validate` method for `createGroupRequest` like we do for other requests, but we can do this in a follow-up.

Reply:
Easy enough, done.